### PR TITLE
CASMINST-4938 Remove SLS Upgrade for 1.3

### PR DESCRIPTION
**Pull request by trad511**
# Description

Remove SLS upgrades for CSM 1.3 - not required. Update Management Network upgrade docs.  Also backports CSM 1.2 SLS changes that allow CHN addresses to be added to computes (CASMNET-1543).

Resolves:
- CASMINST-4938
- CASMINST-4944

# Checklist Before Merging

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/network/management_network/1.2_to_1.3_upgrade.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+## Backup Switch Configurations
+
+- (`ncn-m001#`) Use CANU to backup the switch running configurations.  Enter the switch administrative password when prompted.
+
+```bash
+mkdir switch-upgrades-csm-1.3
+cd switch-upgrades-csm-1.3
+canu backup network --folder 1.2
+```
```
Comment (rsjostrand-hpe) [inline operations/network/management_network/1.2_to_1.3_upgrade.md:31]: This block should probably be indented to be under the bullet point:
```suggestion
  ```bash
  mkdir switch-upgrades-csm-1.3
  cd switch-upgrades-csm-1.3
  canu backup network --folder 1.2
  ```
```

File: operations/network/management_network/1.2_to_1.3_upgrade.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    # CANU version: 1.6.5
+    ###############################################################################
+
+  Login:
+    NVIDIA Onyx Switch Management
+
+  Logout:
+
+sw-spine-001 [mlag-domain: master] (config) # configuration write to csm1.3-canu1.6.5
+```
```
Comment (rsjostrand-hpe) [inline operations/network/management_network/1.2_to_1.3_upgrade.md:74]: Indent this block:
````suggestion
  ```text
  sw-spine-001 [mlag-domain: master] (config) # show banner

  Banners:
    Message of the Day (MOTD):

      ###############################################################################
      # CSM version:  1.2
      # CANU version: 1.6.5
      ###############################################################################

    Login:
      NVIDIA Onyx Switch Management

    Logout:

  sw-spine-001 [mlag-domain: master] (config) # configuration write to csm1.3-canu1.6.5
  ```
````

File: operations/network/management_network/1.2_to_1.3_upgrade.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+## Dell
+
+- Use `canu validate` to see the differences between the 1.2 and 1.3 switch configurations.
+
+```bash
+canu validate switch config --running 1.2/sw-leaf-bmc-001.cfg --generated 1.3/sw-leaf-bmc-001.cfg --vendor dell --remediation
+```
```
Comment (rsjostrand-hpe) [inline operations/network/management_network/1.2_to_1.3_upgrade.md:82]: Indent this block:
````suggestion
  ```bash
  canu validate switch config --running 1.2/sw-leaf-bmc-001.cfg --generated 1.3/sw-leaf-bmc-001.cfg --vendor dell --remediation
  ```
````

File: operations/network/management_network/1.2_to_1.3_upgrade.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+```text
+sw-spine-002(config)# show banner motd
+###############################################################################
+# CSM version:  1.2
+# CANU version: 1.6.5
+###############################################################################
+sw-spine-002(config)# write mem
+Copying configuration: [Success]
+sw-spine-002(config)# copy running-config checkpoint CSM1_3_CANU_1_6_7
+```
```
Comment (rsjostrand-hpe) [inline operations/network/management_network/1.2_to_1.3_upgrade.md:131]: Indent
````suggestion
  ```text
  sw-spine-002(config)# show banner motd
  ###############################################################################
  # CSM version:  1.2
  # CANU version: 1.6.5
  ###############################################################################
  sw-spine-002(config)# write mem
  Copying configuration: [Success]
  sw-spine-002(config)# copy running-config checkpoint CSM1_3_CANU_1_6_7
  ```
````

File: operations/network/management_network/1.2_to_1.3_upgrade.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+  - Changes to ISL configurations.
+  - Changes to Spanning Tree.
+  - Changes to ACLs or ACL ordering.
+  - Changes to `VRFs`.
+  - Changes to default route.
+  - Changes to MLAG/LACP.
+
+- (`m001#`) Using the `remediation config`, apply `prefix-list` and `route-maps` first.
+- (`m001#`) Copy in the remaining configuration.
+- (`m001#`) Save the configuration and create a checkpoint using the CSM version and the CANU version
```
Comment (rsjostrand-hpe) [inline operations/network/management_network/1.2_to_1.3_upgrade.md:120]: As mentioned above, is the this the correct prompt here?

File: operations/network/management_network/1.2_to_1.3_upgrade.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+
```
Comment (rsjostrand-hpe) [inline operations/network/management_network/1.2_to_1.3_upgrade.md:153]: As mentioned above, is the this the correct prompt here?

File: upgrade/scripts/sls/expert_mode.md [hunk 7]
```diff
@@ -35,9 +35,9 @@ The upgrader requires only two inputs:
 An example of this would be:
 
 ```bash
-./sls_updater_csm_1.2.py \
-    --sls-input-file sls_input_file.json \
-    --bican-user-network-name CAN
+ncn# ./sls_updater_csm_1.2.py \
```
Comment (rsjostrand-hpe) [inline upgrade/scripts/sls/expert_mode.md:38]: The command line prompts in this document don't match https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts-inside-of-directions

**Review by rsjostrand-hpe — CHANGES_REQUESTED** [review]

**Review by rustydb — APPROVED** [review]

**Review by lukebates123 — APPROVED** [review]

File: upgrade/scripts/sls/README.SLS_Upgrade.md [hunk 8]
```diff
@@ -35,59 +35,58 @@ Prior to updating SLS, at a minimum, answers to the following questions must be
...
 ## Prerequisites
 
-* The latest CSM documentation RPM must be installed on the node where the procedure is being performed. See
-  [Check for Latest Documentation](../../../update_product_stream/README.md#check-for-latest-documentation).
+* The latest CSM documentation RPM must be installed on the node where the procedure is being performed.
 
 ## Procedure
 
-1. (`ncn#`) Get a token.
+1. Get a token.
```
[the remark this answers is not in the document]
Comment (mharding-hpe) [reply upgrade/scripts/sls/README.SLS_Upgrade.md:42]: This was marked resolved, but was not fixed. I am correcting it as part of this PR:
https://github.com/Cray-HPE/docs-csm/pull/2082

File: upgrade/scripts/sls/expert_mode.md (hunk 7 above)
Comment (mharding-hpe) [reply upgrade/scripts/sls/expert_mode.md:38]: This was marked resolved, but was not fixed. I am correcting it as part of this PR:
https://github.com/Cray-HPE/docs-csm/pull/2082

**Pull request merged**